src/components/Grid/styles.css

```diff
@@ -57,6 +57,16 @@
 
 .grid-cell.t2 {
     background-position: 0px -128px;
+    animation: playerAnim 1s steps(3) infinite;
+}
+
+@keyframes playerAnim {
```

Comment:
This is great!  Does it make sense to also add this to the Tileset window stylesheet?  Then the sprites would be animated in both places.

Reply:
Just pushed the animations for the tileset window